.github/workflows/docker-tests.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         with:
           fetch-depth: 1
       - name: Build Docker image openzipkin/${{ matrix.target }}:test
-        run:  docker/build_image ${{ matrix.target }}
+        run:  docker/build_image openzipkin/${{ matrix.target }}:test
```

Comment:
@odidev when this PR merges, it should be ready to build on arm64. However, I don't know currently how to get the github action syntax to do a double-matrix (ex focal with amd64,arm64 against the docker image list)

Reply:
@adriancole 
Thanks for the PR. Once this is merged we can add in Travis as it is faster for ARM64 with graviton2 in it. If you are ok with this, I can add and raise PR.

Reply:
at the moment we'll still need amd64 though at least on master branch, I think.. because that needs to do multi-arch push now. keep that in mind.

at any rate, yep help appreciated!